packages/nimble-components/src/wafer-map/specs/features/padding.md

```diff
@@ -0,0 +1,38 @@
+# Nimble Wafer Map Die Padding
+
+## Problem Statement
+
+The Wafer Map displays dies on a canvas inside a circular border with a dynamic padding size based on the quantity of dies.
+The padding size needs to be regulated and specific, therefore a public input parameter will be used to allow the setting of padding size.
+
+## Links To Relevant Work Items and Reference Material
+
+[Wafer Map Spec](../README.md)
+
+[D3 ScaleBand](https://observablehq.com/@d3/d3-scaleband)
+
+[D3 ScaleQuantile](https://observablehq.com/@d3/quantile-quantize-and-threshold-scales)
+
+## Implementation / Design
+
+![Die Padding](resources/padding.png)
```

Comment:
It looks like this screenshot is showing what happens with a padding of zero and a hover.
It looks like the size of the die shrinks a bit with hover? Is that what I am seeing in the screenshot or is there some interaction with padding that I don't understand?